module/os/freebsd/zfs/zvol_os.c

```diff
@@ -813,6 +813,64 @@ zvol_geom_bio_strategy(struct bio *bp)
 		biofinish(bp, NULL, error);
 }
 
+static void
+zvol_strategy_task(void *arg)
+{
+	zv_request_task_t *task = arg;
+
+	zvol_strategy_impl(&task->zvr);
+	zv_request_task_free(task);
+}
+
+static void
+zvol_geom_bio_strategy(struct bio *bp, boolean_t sync)
+{
+	zv_taskq_t *ztqs = &zvol_taskqs;
+	zv_request_task_t *task;
+	zvol_state_t *zv;
+	uint64_t taskq_hash;
```

Comment:
`taskq_hash` is `uint_t` intentionally on Linux to avoid 64-bit divisions.

Reply:
Fixed.